x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -171,7 +171,7 @@ export const getSavedObjectTypes = (
       name: AGENT_POLICY_SAVED_OBJECT_TYPE,
       indexPattern: INGEST_SAVED_OBJECT_INDEX,
       hidden: false,
-      namespaceType: useSpaceAwareness ? 'single' : 'agnostic',
+      namespaceType: useSpaceAwareness ? 'multiple' : 'agnostic', // TEMP multiple until we address TODO
```

Comment:
Tmp until this is merged https://github.com/elastic/kibana/pull/189387 same for bellow